### DISCUSSION TITLE
Monospace font family class

### DISCUSSION THREAD

**Discussion by robinsonjohn** [tailwindlabs/tailwindcss#4353] [Ideas]
Current font family classes include `.font-sans` and `.font-serif`. 

What are the thoughts on adding a `.font-mono` or `.font-monospace` class?

**Comment by dance2die** [comment]
There is a `font-mono` for monospaced fonts: https://tailwindcss.com/docs/font-family#monospaced

**Reply by robinsonjohn** [reply]
Wow, I can't believe I missed that somehow. Thank you.
